### ISSUE TITLE
Error on example data

### ISSUE THREAD

**Issue by jenellewallace** [Yunuuuu/eheat#1]
This package looks like exactly what I need but I was not able to reproduce the example. Thanks for any help you can give!

```r
> set.seed(123)
> nr1 <- 4
> nr2 <- 8
> nr3 <- 6
> nr <- nr1 + nr2 + nr3
> nc1 <- 6
> nc2 <- 8
> nc3 <- 10
> nc <- nc1 + nc2 + nc3
> mat <- cbind(
+     rbind(
+         matrix(rnorm(nr1 * nc1, mean = 1, sd = 0.5), nrow = nr1),
+         matrix(rnorm(nr2 * nc1, mean = 0, sd = 0.5), nrow = nr2),
+         matrix(rnorm(nr3 * nc1, mean = 0, sd = 0.5), nrow = nr3)
+     ),
+     rbind(
+         matrix(rnorm(nr1 * nc2, mean = 0, sd = 0.5), nrow = nr1),
+         matrix(rnorm(nr2 * nc2, mean = 1, sd = 0.5), nrow = nr2),
+         matrix(rnorm(nr3 * nc2, mean = 0, sd = 0.5), nrow = nr3)
+     ),
+     rbind(
+         matrix(rnorm(nr1 * nc3, mean = 0.5, sd = 0.5), nrow = nr1),
+         matrix(rnorm(nr2 * nc3, mean = 0.5, sd = 0.5), nrow = nr2),
+         matrix(rnorm(nr3 * nc3, mean = 1, sd = 0.5), nrow = nr3)
+     )
+ )
> mat <- mat[sample(nr, nr), sample(nc, nc)] # random shuffle rows and columns
> rownames(mat) <- paste0("row", seq_len(nr))
> colnames(mat) <- paste0("column", seq_len(nc))
> small_mat <- mat[1:9, 1:9]
> draw(ggheat(small_mat))
Error in `gtable_trim()` at gtable/R/filter.R:42:3:
! `x` must be a gtable object, not a <zeroGrob/grob/gDesc> object.
Run `rlang::last_trace()` to see where the error occurred.
```

**Comment by Yunuuuu** [comment]
Thanks for the report. Should fix by 1ada92a08b5d5a6947108c234ba94d84626365b9

**Comment by jenellewallace** [comment]
Thanks for the fast response, it is working for me now!
